src/ontology/fbbt-edit.obo

```diff
@@ -134584,6 +134623,95 @@ intersection_of: part_of FBbt:00058281 ! adult proventriculus epithelium
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-10-01T12:27:21Z
 
+[Term]
+id: FBbt:00058283
+name: adult ostial cell
+def: "Ostial cell of the adult dorsal vessel. There are 20 of these cells per dorsal vessel, forming 5 pairs of ostia along the dorsal vessel. The 4 cells of the posterior-most pair of ostia are embryonic/larval ostial cells that persist in the adult stage; the other 16 cells differentiate during metamorphosis (Lehmacher et al., 2012; Rotstein and Paululat, 2016)." [FlyBase:FBrf0219200, FlyBase:FBrf0238157]
+subset: cur
+intersection_of: FBbt:00058260 ! ostial cell
+intersection_of: part_of FBbt:00003004 ! adult
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-12-08T10:46:11Z
+
+[Term]
+id: FBbt:00058284
+name: embryonic/larval ostial cell
+def: "Ostial cell of the embryonic/larval dorsal vessel. There are 12 of these cells per dorsal vessel, forming in total 3 pairs of ostia in the posterior heart chamber of the late embryo and larva. During embryogenesis, the progenitors of these cells are distinguished from the other cardiomyocytes by the specific expression of Svp and the absence of expression of tinman. During metamorphosis, the 8 cells forming the two posterior-most pairs are hystolized and only the 4 cells of the anterior-most pair persist in the adult stage (Lehmacher et al., 2012; Rotstein and Paululat, 2016)." [FlyBase:FBrf0219200, FlyBase:FBrf0238157]
+subset: cur
+intersection_of: FBbt:00058260 ! ostial cell
+intersection_of: part_of FBbt:00001727 ! larva
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-12-08T10:51:33Z
+
+[Term]
+id: FBbt:00058285
+name: intracardial valve cell
+def: "A specialized cardiomyocyte that forms a valve inside the dorsal vessel. There are two such cells per valve, located contralaterally. They have large intracellular vesicles that occupy most of their volume, a high number of mitochondria, and a network-like arrangement of myofilaments (Lehmacher et al., 2012; Rotstein and Paululat, 2016)." [FlyBase:FBrf0219200, FlyBase:FBrf0238157]
+subset: cur
+is_a: FBbt:00007319 ! cardial cell
+relationship: part_of FBbt:00058288 ! intracardial valve
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-12-08T11:02:46Z
+
+[Term]
+id: FBbt:00058286
+name: adult intracardial valve cell
+def: "Intracardial valve cell of the adult dorsal vessel. There are 6 such cells per dorsal vessel, forming 3 valves separating the 4 chambers of the adult heart (Lehmacher et al., 2012; Rotstein and Paululat, 2016)." [FlyBase:FBrf0219200, FlyBase:FBrf0238157]
+subset: cur
+intersection_of: FBbt:00058285 ! intracardial valve cell
+intersection_of: part_of FBbt:00058289 ! adult intracardial valve
```

Comment:
This and the larval version should follow the same pattern of \<stage\> \<anatomy\> == \<anatomy\> that part_of some \<stage\>
With a part_of relationship to the stage-specific valve.